### DISCUSSION TITLE
develop a connector to DBT

### DISCUSSION THREAD

**Discussion by tomerbm** [dbt-labs/dbt-core#6160] [Community]
I am looking for someone to help develop a connector to DBT , something like https://github.com/MaterializeInc/materialize/tree/main/misc/dbt-materialize

**Comment by lostmygithubaccount** [comment]
hi @tomerbm, there's a thorough guide here on developing new adapters: https://docs.getdbt.com/guides/advanced/adapter-development/1-what-are-adapters

what are you looking to developer an adapter for?
